### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by diegoh40**
In this part of the project with my programming partner we were able to fix bugs in the projects for a while and thus make contributions or participations in their code in this way we learned to work as a team using the controller and browser model to fix and improve parts of the code.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -106,47 +110,108 @@ <h1>About me</h1>
...
               <li>Github</li>
               <li>Codepen</li>
               <li>Gitlab</li>
               <li>Terminal</li>
+            </ul>
+          </div>
+        </div>
+      </section>
+      <section class="form">
+        <div>
```
Comment (mahbubul14) [inline index.html:159]: - [ ] Currently, there's a horizontal scroll bar on your page. Kindly fix it. You can use [CSS Media Queries](https://www.w3schools.com/css/css3_mediaqueries.asp) to make your page responsive well. :writing_hand: 

https://user-images.githubusercontent.com/70483835/165421027-54cbfa1f-5136-4ac9-b051-89b0356b0b53.mp4

File: index.html [hunk 2]
```diff
@@ -106,47 +110,108 @@ <h1>About me</h1>
...
+          <h6>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h6>
```
Comment (mahbubul14) [inline index.html:163]: - [ ] You `h6` text is not matched with the Figma design. In the Figma design, there is more white space on the both left and right sides. Please work on it.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
 |

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @diegoh40 ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] In your `README` file, there is no "Getting Started" section. Kindly add the getting started section. This should be an example of how you may give instructions on setting up your project locally.

- [ ] Please check the inine comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [ ] The commit message should be written in the imperative tense and the first letter will be capitalized. `Ex: "Fix bug" instead of "Fixed bug" /"fix bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

- [ ] Your mobile version of the app is not currently optimized well. You can use [CSS Media Queries](https://www.w3schools.com/css/css3_mediaqueries.asp) to make your page responsive well. Please work on it. :writing_hand: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by diegoh40** [comment]
In this new update with respect to the suggestions of the code reviewer was added the startup explaining the instructions, improvements were made to make the page responsive, design details were corrected in the Css, Html tags were simplified to make them easier to understand and execute.

**Review by aimemalaika — APPROVED** [review]
Hi @diegoh40, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**